prime-router/src/main/kotlin/Element.kt

```diff
@@ -55,6 +55,7 @@ data class Element(
     val maxLength: Int? = null, // used to truncate outgoing formatted String fields.  null == no length limit.
     val default: String? = null,
     val mapper: String? = null,
+    val mapperAlwaysRun: Boolean = false,
```

Comment:
The use case for this is to override an existing value, the name can make that clear. Something that makes it clear that the mapper will `override` any existing value

Reply:
How about mapperOverridesValue?